x-pack/plugins/data_enhanced/public/search/ui/connected_search_session_indicator/search_session_tour.tsx

```diff
@@ -79,15 +80,3 @@ function safeSet(storage: IStorageWrapper, key: string) {
     return true;
   }
 }
-
-function safeOpen(searchSessionIndicatorRef: MutableRefObject<SearchSessionIndicatorRef | null>) {
-  if (searchSessionIndicatorRef.current) {
-    searchSessionIndicatorRef.current.openPopover();
-  } else {
-    // TODO: needed for initial open when component is not rendered yet
-    // fix after: https://github.com/elastic/eui/issues/4460
-    setTimeout(() => {
```

Comment:
Some of the code is this pr (changes related to `searchSessionIndicatorRef`) is to remove this hack.
In future, we can simplify this, but we'd need EUI version with https://github.com/elastic/eui/issues/4460 fix in it.